### PR TITLE
Open OpenFlow port for SDN controller

### PR DESCRIPTION
**Pull request by benjamreis**

### CODE REVIEW

**Comment by benjamreis** [comment]
build & xapi tests ok

File: SPECS/xapi.spec [hunk 1]
```diff
@@ -438,6 +439,10 @@ Coverage files from unit tests
 %endif
 
 %changelog
+* Fri Apr 24 2020 Benjamin Reis <benjamin.reis@vates.fr> - 1.160.2-1.3
+- Patch xapi-1.160.2-open-openflow-port.XCP-ng.patch modified
```
Comment (stormi) [inline SPECS/xapi.spec:443]: Patch added, not modified?

Comment (benjamreis) [reply SPECS/xapi.spec:443]: right, i might have made the same mistake on 8.1...

Comment (benjamreis) [reply SPECS/xapi.spec]: just pushed a fix. I'll create a PR fixing 8.1

**Pull request merged**